src/e2e/specs/auth.spec.ts

```diff
@@ -41,12 +38,8 @@ test.describe(`${process.env.E2E_TEST_ENV} - Authentication flow verification @s
     // wait for scanning to complete
     let wait = 0;
     let scanning = page.url() !== successUrl;
-    while (wait < 60 && scanning) {
-      try {
-        scanning = page.url() !== successUrl;
-      } catch (error) {
-        console.log("line 45");
-      }
+    while (wait < 300 && scanning) {
```

Comment:
What's this `wait < 300`?
I see we init to `0` above on L39, and increment on L44, but with a 1s pause, that's 300s or 5 minutes of potential pausing before the test fails?
Why the 5&times; increase from 60s timeout in the old code?

Reply:
this PR is pretty much a debugging exercise, there's a redirect issue that i'm looking at, will clean this up once we get to a solid solution